vpr/src/route/route_common.cpp

```diff
@@ -1383,6 +1372,8 @@ void reserve_locally_used_opins(float pres_fac, float acc_fac, bool rip_up_local
         }
     }
 
+    heap->empty_heap();
```

Comment:
It's not clear why this is needed here, so we should have a comment describing why.

Reply:
Done

Reply:
Done